requirements/core.txt

```diff
@@ -6,7 +6,7 @@
 cartopy
 #conda: proj4<5
 cf_units>=2
-cftime!=1.0.2.1
+cftime<1.0.2.1
```

Comment:
Why not pin directly to `1.0.1`, just to make this explicit and clear.

We still need to fix `iris` to deal with 0 based dates, which are now illegal in the latest `cftime`.

So this pin would only be a temporary pin...

Reply:
> ...Why not pin directly...

Sure.

> We still need to fix iris to deal with 0 based dates...

Agreed.  Some relating discussion in #3239